### ISSUE TITLE
[Task]: Automated accessibility testing mechanism

### ISSUE THREAD

**Issue by acouch** [HHS/simpler-grants-gov#764]
### Summary

There is a requirement in the #58 for automated accessibility testing. 

Personally I've used [pa11y](https://pa11y.org/) in the past. It was easy to setup and test locally and in the CI.

There are several steps to this task:

- [ ] Decide on tool, pa11y or other. Can poll internally or elsewhere if helpful.
- [ ] Confirm spec to test against. Currently believe that is 508.

### Acceptance criteria

- [ ] Automated accessibility test tool decided and documented

### Out of scope
- We want to have a mechanism for recurring accessibility testing. We need to determine what should be the right approach. There should be an ADR in the future to consider this

### Notes
- This doesn't remove the need to have a 508 compliance review. We could consider chatting with the 508 team to understand how they are doing the tests.  
- Consider time-boxing this ticket so we limit the complexity

**Comment by SammySteiner** [comment]
We're already doing accessibility testing with Axe. Maybe just document it in frontend/readme.md in the testing section.

**Comment by acouch** [comment]
This was completed: #2077
